backend/index.js

```diff
@@ -0,0 +1,17 @@
+import OfficeController from "./controllers/office.controller";
+import Office from "./office.server";
+import app from "./app.server";
+
+const PORT = process.env.PORT || 8080;
+const HOST = "0.0.0.0";
+
+const server = app.listen(PORT, HOST, undefined, () => {
+  console.log(`Running on http://${HOST}:${PORT}`);
+});
+
+const officeControllerInstance = new OfficeController();
+const office = new Office(officeControllerInstance, server);
```

Comment:
Maybe this source can stay inside `backend/socket.js`, to separate routes and socket events, and keep this file only to app boot.

Reply:
I changed and explained in the other review.